data/birth-dates/19.txt

```diff
@@ -1 +1 @@
-Sophie Wilson (born Roger Wilson): ????
```

Comment:
@swagner-de I have a different birth day. Where do you have this information form?